ibis/backends/pyspark/__init__.py

```diff
@@ -662,8 +686,13 @@ def read_parquet(
             The just-registered table
 
         """
-        source = util.normalize_filename(source)
-        spark_df = self._session.read.parquet(source, **kwargs)
+        if self.mode == "streaming":
+            raise NotImplementedError(
+                "Pyspark in streaming mode does not support direction registration of parquet files. "
+                "Please use `read_parquet_directory` instead."
```

Comment:
We don't technically have `read_parquet_directory` implemented yet but I figured that it may be okay because it's going in in a separate PR? But I can delete this if needed.